bundles/framework/layerlist/view/LayerViewTabs/SelectedLayers/SelectedLayersHandler.js

```diff
@@ -54,23 +72,28 @@ class UIService extends StateHandler {
 
     changeOpacity (layer, opacity) {
         this.sandbox.postRequestByName('ChangeMapLayerOpacityRequest', [layer.getId(), opacity]);
-        this.updateState({ layers: this._getLayers() });
+        this.updateLayers();
     }
 
     removeLayer (layer) {
         this.sandbox.postRequestByName('RemoveMapLayerRequest', [layer.getId()]);
-        this.updateState({ layers: this._getLayers() });
+        this.updateLayers();
```

Comment:
This is redundant because `LayerViewTabsHandler` listens for `AfterMapLayerRemovedEvent` and refreshes the ui.